Source/WebCore/style/values/backgrounds/StyleLineWidth.h

```diff
@@ -77,7 +78,7 @@ template<typename Result> struct Evaluation<LineWidth, Result> {
         if (auto minimumLineWidth = 1.0f / zoom.deviceScaleFactor; result > 0.0f && result < minimumLineWidth)
             return Result(minimumLineWidth);
 
-        return Result(floorToDevicePixel(result, zoom.deviceScaleFactor));
+        return Result(floorf(result * zoom.deviceScaleFactor) / zoom.deviceScaleFactor);
```

Comment:
I think we prefer:

`return Result(std::floor(result * zoom.deviceScaleFactor) / zoom.deviceScaleFactor);`